crates/biome_cli/Cargo.toml

```diff
@@ -73,8 +71,7 @@ regex                = { workspace = true }
 tokio                = { workspace = true, features = ["io-util"] }
 
 [features]
-docgen     = ["bpaf/docgen"]
-format_css = ["biome_css_formatter/format_css"]
```

Comment:
This is a bug in `shear`, we need to keep it. Thank you :)

Reply:
fixed at [ac7a9cc](https://github.com/biomejs/biome/pull/2153/commits/ac7a9cc8f44cf6c75d1d2e05821c78e8e5d11c1f)!